spec/lib/tasks/user_spec.rb

```diff
@@ -30,7 +34,13 @@
       expect(user.is_not_approved?).to be true
     end
 
+    it 'leaves approved users with sites to approved' do
+      @rake[task_name].invoke
+      expect(user_with_one_site.reload.is_approved?).to be true
+    end
+
     it 'sends admin the user_approval_removed email' do
+      User.destroy_all("email != 'affiliate_manager_with_no_affiliates@fixtures.org'")
```

Comment:
Is this still necessary?

Reply:
users who don't have sites get an email when ther user approval is removed. It was what was on prod. I would think the test is necessary as we still send email yes.

Reply:
Oh if you mean the destroy_all yes still needed.

Reply:
👍